src/liboslexec/batched_llvm_gen.cpp

```diff
@@ -4054,8 +4054,8 @@ LLVMGEN (llvm_gen_calculatenormal)
 
     llvm::Value *args[] = {
         rop.llvm_void_ptr (Result),
-        rop.sg_void_ptr(),
         rop.llvm_load_arg (P, true /*derivs*/, false /*op_is_uniform*/),
+        rop.llvm_global_symbol_ptr(Strings::flipHandedness),
```

Comment:
As a matter of sanity checking, might consider  pulling the global_symbol_ptr out and passing the optional is_uniform parameter and OSL_ASSERT(!is_uniform) as the function being called will interpret the pointer as varying.